docs/resources/Guild.md

```diff
@@ -785,17 +777,46 @@ Sync an integration. Requires the `MANAGE_GUILD` permission. Returns a 204 empty
 
 Returns the [guild widget](#DOCS_RESOURCES_GUILD/guild-widget-object) object. Requires the `MANAGE_GUILD` permission.
 
-## Get Guild Embed % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/embed
-
-Same as above, but this endpoint is deprecated.
-
 ## Modify Guild Widget % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/widget
 
 Modify a [guild widget](#DOCS_RESOURCES_GUILD/guild-widget-object) object for the guild. All attributes may be passed in with JSON and modified. Requires the `MANAGE_GUILD` permission. Returns the updated [guild widget](#DOCS_RESOURCES_GUILD/guild-widget-object) object.
 
-## Modify Guild Embed % PATCH /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-object}/embed
+## Get Guild Widget JSON % GET /guilds/{guild.id#DOCS_RESOURCES_GUILD/guild-bject}/widget.json
```

Comment:
1. bject should be object
2. I would name this just "Get Guild Widget" and rename the other routes to "... Widget Settings" (`get_widget_settings` internally)